### DISCUSSION TITLE
How did I can set the space color of Meter as same as the container Frame?

### DISCUSSION THREAD

**Discussion by syo-ki** [israel-dryer/ttkbootstrap#376] [Q&A]
![image](https://user-images.githubusercontent.com/105769349/204718186-26586620-8fd4-43eb-be2c-d65aae8f57dd.png)

As the image showed, I've put a Meter on the Frame container which bootstyle="light". How can I set the background space color of darkgrey? I tried to adjusted the "bootstyle.Meter.TLabel" style configuration but It only work on the background color of label.
![image](https://user-images.githubusercontent.com/105769349/204719377-ecee503a-d689-4fba-be60-283559ccab5d.png)

**Comment by botkero** [comment]
Can you show more of your code? I had a similar problem. For me, it helped to take out the bootstyle in certain places.

**Comment by TheDemonGuard** [comment]
@syo-ki 

I cannot reproduce the issue with my following sample code with the latest ttkbootstrap version (1.18.0).
Is it possible to try with the latest version.
If the issue still persists, please modify the sample code accordingly and/or provide your own demo/sample code for investigation purposes.

<img width="237" height="308" alt="image" src="https://github.com/user-attachments/assets/3e03c95e-5822-4a96-89e7-45014947f0d4" />

 
```
    import ttkbootstrap as ttk
    from ttkbootstrap.constants import *

    app = ttk.Window()

    frame = ttk.Frame(app, bootstyle="light")
    meter = ttk.Meter(
        master=frame,
        metersize=180,
        padding=5,
        amountused=25,
        metertype="semi",
        subtext="miles per hour",
        interactive=True,
        bootstyle="success",
        stripethickness =4,
        subtextstyle="dark"
    )
    meter.pack()
    frame.pack()
    # update the amount used directly
    meter.configure(amountused = 50)

    # update the amount used with another widget
    entry = ttk.Entry(textvariable=meter.amountusedvar)
    entry.pack(fill=X)

    # increment the amount by 10 steps
    meter.step(10)

    # decrement the amount by 15 steps
    meter.step(-15)

    # update the subtext
    meter.configure(subtext="loading...")

    app.mainloop()
```


> ![image](https://user-images.githubusercontent.com/105769349/204718186-26586620-8fd4-43eb-be2c-d65aae8f57dd.png)
> 
> As the image showed, I've put a Meter on the Frame container which bootstyle="light". How can I set the background space color of darkgrey? I tried to adjusted the "bootstyle.Meter.TLabel" style configuration but It only work on the background color of label. ![image](https://user-images.githubusercontent.com/105769349/204719377-ecee503a-d689-4fba-be60-283559ccab5d.png)
